wagtail/wagtailsearch/index.py

```diff
@@ -203,6 +203,8 @@ def get_value(self, obj):
         try:
             field = self.get_field(obj.__class__)
             value = field.value_from_object(obj)
+            if hasattr(self, 'search_content') and self.search_content:
```

Comment:
So: going back to https://github.com/torchbox/wagtail/pull/2992/commits/96ddb64ffb9f5b5a26914c9ba909d594930ffb92#diff-7ba5f4d8cfc78dcc93e806ec45f75d4fR29 ?